README.rst

```diff
@@ -141,6 +141,35 @@ To change the MFEs logos from the default to your own logos, override the corres
         LOGO_WHITE_URL=<URL>/logo-white.svg
         FAVICON_URL=<URL>/favicon.ico
 
+To install custom components for the MFEs, such as the `header <https://github.com/openedx/frontend-component-header>`_ and `footer <https://github.com/openedx/frontend-component-footer>`_, override the components by adding a patch to ``mfe-dockerfile-post-npm-install`` in your plugin:
+::
+
+    patches:
+        ...
+        mfe-dockerfile-post-npm-install |
```

Comment:
Isn't a ":" missing here?

```
mfe-dockerfile-post-npm-install: |
    ...
```

Same comment for the patches below.

Reply:
Ah, yes, I'll fix that, thanks!